lib/generators/simple_form/templates/config/initializers/simple_form_bootstrap.rb

```diff
@@ -126,7 +126,7 @@
     b.use :html5
     b.optional :readonly
     b.use :label, class: 'control-label'
-    b.wrapper tag: 'div', class: 'form-inline' do |ba|
+    b.wrapper tag: 'div', class: 'form-inline col-sm-10' do |ba|
```

Comment:
Yeah, it's just affect horizontal form I think.

I use col-sm-10 since I saw it on the other form element.

If this minor addition shouldn't located here, we should add new option to
change the "<div inline form>" class

Thanks.